src/Drawer/Drawer.js

```diff
@@ -41,7 +41,8 @@ export const styleSheet = createStyleSheet('MuiDrawer', (theme) => {
       WebkitOverflowScrolling: 'touch', // Add iOS momentum scrolling.
     },
     docked: {
-      flex: '0 0 auto',
```

Comment:
No, what I mean is that this change is a good one IMHO, but we need to address the side effects.